### PR TITLE
[GG 2486] Upload poi mbtiles to mapbox account

### PR DESCRIPTION
**Pull request by sshalu12**
Upload mbtiles to mapbox account

### CODE REVIEW

File: Dockerfile-python [hunk 1]
```diff
@@ -2,11 +2,10 @@ FROM python:3.6
...
 RUN pip install -r /requirements.txt
 RUN apt update
-RUN apt install -y postgresql 
 
-RUN apt-get update \
-  && apt-get -y upgrade \
-  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev
+RUN apt install -y postgresql \
+  && apt -y upgrade \
+  && apt -y install git build-essential libsqlite3-dev zlib1g-dev
```
Comment (ashutoshgt) [inline Dockerfile-python]:
```suggestion
  && apt -y install git build-essential libsqlite3-dev zlib1g-dev postgresql
```

File: Dockerfile-python [hunk 2]
```diff
@@ -2,11 +2,10 @@ FROM python:3.6
 COPY requirements.txt /
 RUN pip install -r /requirements.txt
 RUN apt update
-RUN apt install -y postgresql 
 
-RUN apt-get update \
-  && apt-get -y upgrade \
-  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev
+RUN apt install -y postgresql \
```
Comment (ashutoshgt) [inline Dockerfile-python]: remove this line

File: script/poi.py [hunk 3]
```diff
@@ -19,6 +22,10 @@
 query=cur.fetchall()
```
Comment (ashutoshgt) [inline script/poi.py]:
```suggestion
poi_data=cur.fetchall()
```

File: script/poi.py [hunk 4]
```diff
@@ -19,6 +22,10 @@
...
 conn.close()
 
+if(len(query)==0):
```
Comment (ashutoshgt) [inline script/poi.py]:
```suggestion
if len(query)==0:
```

File: script/poi.py [hunk 5]
```diff
@@ -19,6 +22,10 @@
...
+    print("No data fetched from database ")
```
Comment (ashutoshgt) [inline script/poi.py]: use logging module here

File: script/poi.py [hunk 6]
```diff
@@ -62,3 +65,21 @@
     mbtiles_file='./app/poi.mbtiles',
     maxzoom=10
 )
+
+service= Uploader()
```
Comment (ashutoshgt) [inline script/poi.py]:
```suggestion
service = Uploader()
```

File: script/poi.py [hunk 7]
```diff
@@ -62,3 +65,21 @@
...
+mapid = "poi"
+
+with open('app/poi.mbtiles', 'rb') as src:
+    upload_resp = service.upload(src, mapid)
+
+    """ response status code 422 indicates that the server understands the content type of the request entity,
+     and the syntax of the request entity is correct, but it was unable to process the contained instructions. To overcome this problem 
+    script in sleep mode for 5 second and then retry"""
```
Comment (ashutoshgt) [inline script/poi.py]:
```suggestion
    """ 
      Response status code 422 indicates that the server understands the content type of the request entity,
      and the syntax of the request entity is correct, but it was unable to process the contained instructions. To 
      overcome this problem put the script in sleep mode for 5 seconds and then retry.
     """
```

**Review by ashutoshgt — CHANGES_REQUESTED** [review]

File: Dockerfile-python (hunk 1 above)
Comment (sshalu12) [reply Dockerfile-python]: Done

File: Dockerfile-python (hunk 2 above)
Comment (sshalu12) [reply Dockerfile-python]: Done

File: script/poi.py (hunk 3 above)
Comment (sshalu12) [reply script/poi.py]: Done

File: script/poi.py (hunk 4 above)
Comment (sshalu12) [reply script/poi.py]: Done

File: script/poi.py (hunk 6 above)
Comment (sshalu12) [reply script/poi.py]: Done

File: script/poi.py (hunk 7 above)
Comment (sshalu12) [reply script/poi.py]: Done

File: script/poi.py [hunk 8]
```diff
@@ -62,3 +66,23 @@
...
+mapid = "poi"
+
+with open('app/poi.mbtiles', 'rb') as src:
+    upload_resp = service.upload(src, mapid)
+
+    """ 
+    response status code 422 indicates that the server understands the content type of the request entity,
+    and the syntax of the request entity is correct, but it was unable to process the contained instructions.
+    To overcome this problem script in sleep mode for 5 second and then retry
+    """
```
Comment (ashutoshgt) [inline script/poi.py]: there's an extra indentation here, please shift this to the left

**Review by ashutoshgt — CHANGES_REQUESTED** [review]

**Review by ashutoshgt — APPROVED** [review]

**Pull request merged**